doc/source/reference/internals.code-explanations.rst

```diff
@@ -460,12 +460,12 @@ Ufuncs allow other array-like classes to be passed seamlessly through
 the interface in that inputs of a particular class will induce the
 outputs to be of that same class. The mechanism by which this works is
 the following. If any of the inputs are not ndarrays and define the
-:obj:`__array_wrap__` method, then the class with the largest
-:obj:`__array_priority__` attribute determines the type of all the
+:obj:`~numpy.class.__array_wrap__` method, then the class with the largest
```

Comment:
Ok, that's pretty weird! Still, I guess if this PR is all about fixing links, then this is the right thing to do